static/js/search_suggestion.js

```diff
@@ -162,6 +162,80 @@ function get_private_suggestions(all_people, operators, person_operator_matches)
     return suggestions;
 }
 
+function get_group_suggestions(all_people, operators) {
+    if (operators.length === 0) {
+        return [];
+    }
+
+    if ((operators[0].operator === 'is') && (operators[0].operand === 'private')) {
+        operators = operators.slice(1);
+    }
+
+    if (operators.length !== 1 || operators[0].operator !== 'pm-with') {
+        return [];
+    }
```

Comment:
Technically we don't have strong guarantees about ordering of operators -- one can type any combination, in whatever order one wants.  So I'm not thrilled with this initial part of the function, but that's also how some of the adjacent ones work already.  I think I'm OK with merging this and leaving cleaning that up to a follow-up issue, since this adds good tests for the feature and thus we can iterate on this later.